### DISCUSSION TITLE
Curated release notes | multiple release channels

### DISCUSSION THREAD

**Discussion by pat-s** [quarto-dev/quarto-cli#1038] [Q&A] [answered]
Currently quarto is heavy development and new versions are released daily.

I was wondering whether the team is planning to have some curated release notes or introducing different release channels?
E.g. instead of going with the "daily" builds, I could see the desire for a "stable" channel which is only updated once a week/month and bundles release notes for that period.

**Comment by jjallaire** [comment, marked as answer]
Yes, indeed we are! Our currently plan is to have a v1.0 stable release sometime this quarter and then ship stable point releases every few weeks or months after that (there will always still be a daily build channel). We are starting our wind down on v1.0 right and we'll be progressively decelerating development on `main` until that release is finalized.
